index.html

```diff
@@ -1,3 +1,64 @@
-﻿
+<!doctype html>
+<html lang="en">
+  <head>
+    <meta name="description" content="Grandma's Famous Cookie Recipe">
 
+    <title>Cookies</title>
+  </head>
 
+  <body>
+    <img src="./img/cookies.jpg">
+    <header>
+      <h1>The Best Chocolate Chip Cookies</h1>
+    </header>
+
+    <section>
+      <p>Recipe by Grandma</p>
+      <p>Prep Time: 45 Min</p>
+    </section>
+
+    <section>
+      <h2>
+        INGREDIENTS
+      </h2>
+      <ul>
+        <li>1 1/2 cup (3 sticks) softened butter</li>
+        <li>1 cup brown sugar</li>
+        <li>1 cup granulated sugar</li>
+        <li>1 Tbl vanilla instant pudding powder</li>
+        <li>2 Tbl milk</li>
+        <li>2 Tbl vanilla extract</li>
+        <li>2 eggs</li>
+        <li>4 cups all purpose flour</li>
+        <li>2 tsp baking soda</li>
+        <li>1/2 tsp salt</li>
+        <li>4 cups chocolate chips</li>
+        <li>1 cup chopped walnuts or pecans(optional)</li>
+      </ul>
+      </section>
+      <section>
+      <h2>
+        How to make Mama’s Recipe:
+      </h2>
+        <ol type="1">
+          <li>Preheat oven to 350 degrees Beat butter and sugars together until light and fluffy.</li>
+          <li>Stir in pudding mix milk and vanilla extract.</li>
+          <li>Beat in eggs.</li>
+          <li>Add dry ingredients and stir until combined.</li>
+          <li>Stir in chocolate chips and nuts.</li>
+          <li>Place 1 1/2 inch balls of dough 2 inches apart on an un greased cookie sheet.</li>
+          <li>Bake 8-10 minutes or until golden brown.</li>
+        </ol>
+      </section>
+      <h2>
+        Nutrition Information
+      </h2>
+      <p>
+        Probably bad for you
+      </p>
+      <a href=""http://www.opensourcefood.com/people/Amanori/recipes/mamas-recipe-the-best-chocolate-chip-cookies"">Inspiration Recipe</a>
```

Comment:
The double double quotes are what's preventing the link from working. Try `"` instead of `""`.